### ISSUE TITLE
Cannot find module 'minimist'

### ISSUE THREAD

**Issue by clemisch** [AkshaySG14/haystack-public#1]
I'm trying to run Haystack on Linux (Fedora 40). It gives me this error:

```
$ ./haystack-editor
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'minimist'
Require stack:
- /home/clem/Downloads/Haystack+Editor+Linux/Haystack/resources/app/out/main.js
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:1055:15)
    at s._resolveFilename (node:electron/js2c/browser_init:2:115690)
    at Module._load (node:internal/modules/cjs/loader:908:27)
    at c._load (node:electron/js2c/node_init:2:13672)
    at Module.require (node:internal/modules/cjs/loader:1122:19)
    at require (node:internal/modules/helpers:130:18)
    at parseCLIArgs (/home/clem/Downloads/Haystack+Editor+Linux/Haystack/resources/app/out/main.js:8:2042)
    at Object.<anonymous> (/home/clem/Downloads/Haystack+Editor+Linux/Haystack/resources/app/out/main.js:7:8365)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1311:10)
    at Module.load (node:internal/modules/cjs/loader:1098:32)
    at Module._load (node:internal/modules/cjs/loader:945:12)
    at c._load (node:electron/js2c/node_init:2:13672)
    at node:electron/js2c/browser_init:2:121151
    at node:electron/js2c/browser_init:2:121360
    at node:electron/js2c/browser_init:2:121364
    at BuiltinModule.compileForInternalLoader (node:internal/bootstrap/realm:392:7)
    at BuiltinModule.compileForPublicLoader (node:internal/bootstrap/realm:328:10)
    at loadBuiltinModule (node:internal/modules/helpers:68:7)
    at Module._load (node:internal/modules/cjs/loader:923:17)
    at c._load (node:electron/js2c/node_init:2:13672)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:96:12)
    at node:internal/main/run_main_module:23:47
```

I tried searching for `Cannot find module 'minimist'` and VSCode but could't find suitable answers. `$ npm install minimist` didn't help.

**Comment by AkshaySG14** [comment]
Sorry I completely missed this. Could you try `npm link minimist` or alternatively maybe installing it globally? Happy to jump on a call with you to sort this out.

**Comment by clemisch** [comment]
Thanks for the feedback. 

`$ npm link minimist` (in `~` or Haystack dir) gives me a permission error. 

`$ sudo npm link minimist` runs but the error with Haystack is unchanged.

Sorry, I have no previous experience with npm at all. I would assume I'm not alone though so it would be great to solve this!

**Comment by AkshaySG14** [comment]
Are you in the Discord (https://discord.gg/apFrN6ABxc)? I'd love to hop in a call with you to solve this.

**Comment by clemisch** [comment]
It seems to have fixed itself with version 0.15
